tools/esp8266/hmDefines.h

```diff
@@ -20,16 +20,17 @@ union serial_u {
 enum {UNIT_V = 0, UNIT_A, UNIT_W,  UNIT_WH, UNIT_KWH, UNIT_HZ, UNIT_C, UNIT_PCT, UNIT_VAR, UNIT_NONE};
 const char* const units[] = {"V", "A", "W", "Wh", "kWh", "Hz", "°C", "%", "var", ""};
 
-
 // field types
 enum {FLD_UDC = 0, FLD_IDC, FLD_PDC, FLD_YD, FLD_YW, FLD_YT,
         FLD_UAC, FLD_IAC, FLD_PAC, FLD_F, FLD_T, FLD_PF, FLD_EFF,
-        FLD_IRR, FLD_Q,FLD_ALARM_MES_ID,FLD_FW_VERSION,FLD_FW_BUILD_YEAR,
-        FLD_FW_BUILD_MONTH_DAY,FLD_HW_ID,FLD_ACT_PWR_LIMIT,FLD_LAST_ALARM_CODE};
+        FLD_IRR, FLD_Q, FLD_EVT, FLD_FW_VERSION, FLD_FW_BUILD_YEAR,
+        FLD_FW_BUILD_MONTH_DAY, FLD_FW_BUILD_HOUR_MINUTE, FLD_HW_ID,
+        FLD_ACT_ACTIVE_PWR_LIMIT, /*FLD_ACT_REACTIVE_PWR_LIMIT, FLD_ACT_PF,*/ FLD_LAST_ALARM_CODE};
```

Comment:
Why are the two fields reactive Power Limit and active Power Factor undefined / commented out ?

Reply:
Ist stand jetzt nur eine sehr wahrscheinliche Vermutung, aber da das setzen der beiden bisher noch nicht geklappt hat, kann es nicht gegengeprüft werden. Muss dann später mal nur noch auskommentiert werden und ist soweit vorbereitet.

Reply:
Sollte in fields[] auch noch auskommentiert werden, habe ich vergessen :-/

Reply:
Muss es auskommentiert werden?